Apps/Inc/fifo.h

```diff
@@ -42,6 +42,8 @@
 #ifndef __FIFO_H
 #define __FIFO_H
 #include <stdbool.h>
+#include <string.h>
```

Comment:
are these necessary? Fifo.h has been working for a long time. again, not rly a problem, just curious as to why

Reply:
I thought I removed this. Will fix.